driver-core/src/main/com/mongodb/client/model/mql/MqlExpression.java

```diff
@@ -953,6 +953,15 @@ public MqlString asString(final MqlString timezone, final MqlString format) {
                 .append("timezone", toBsonValue(cr, timezone))));
     }
 
+    @Override
+    public MqlString asString(final MqlString timezone) {
```

Comment:
I believe that we do not need this method, and that it might actually be introducing the bug described in the Jira. That is, the API as it is now intentionally prevents the user from specifying a timezone without also forcing that user to explicitly specify a format, thus preventing the situation from arising.

For reference:
* https://github.com/mongodb/mongo-java-driver/pull/1045#discussion_r1028655799
* https://jira.mongodb.org/browse/SERVER-74002 (not as closely related)

Reply:
thanks for sharing. Yeah, also saw you raising the point in our team slack channel. I have no objection of closing this PR.